tbclient/natRule.go

```diff
@@ -112,3 +112,85 @@ func (c *Client) DeleteIpNatRule(uid string) error {
 
 	return executeDelete(c.createRestClient(), c.Token, url)
 }
+
+func (c *Client) GetAllVmNatRules(topologyUid string) ([]VmNatRule, error) {
```

Comment:
It was not possible to create a generic service for this due to the real instantiation of either VM Nat Rule or IP Nat Rule, which can't be implemented in generic code in Go